packages/utils/src/index.ts

```diff
@@ -40,3 +40,4 @@ export * from './typedEventEmitter';
 export * from './urlToOnion';
 export * from './logs';
 export * from './logsManager';
+export * from './bigNumber';
```

Comment:
maybe not exporting this via index could help prevent some accidental unwanted bundling of it? just thinking

Reply:
Yeah, we can definitely do that, would that solve things?

Reply:
not sure, lets summon @Nodonisko

Reply:
Do not overcomplicate things, I thinked it though and it's probably fine to keep it as it is. We already have this dependency in nearly every package anyway and no-one uses out utils lib without rest of our libs.